build_tools/pypi_deploy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This script promotes Python packages from nightly releases to PyPI.
+#
+# Prerequisites:
+#   * For deploying to PyPI, you will need to have credentials set up. See
+#     https://packaging.python.org/en/latest/tutorials/packaging-projects/#uploading-the-distribution-archives
+#     (Googlers can also access the shared releasing account "google-iree-pypi-deploy"
+#     at http://go/iree-pypi-password)
```

Comment:
```suggestion
```

Reply:
I thought this would technically still be true, but https://pypi.org/user/google-iree-pypi-deploy/ isn't actually a member for https://pypi.org/project/iree-turbine/ (and the invite expired for that account on the iree-base-compiler project x_x)